sales_and_marketing.md

```diff
@@ -0,0 +1,38 @@
+# Sales and marketing
+
+- **Sales**: closing deals and generating revenue through direct interaction with customers.
+- **Marketing**: a broader approach, creating awareness, building relationships, and driving long-term sales growth.
+
+This page focuses on the latter with it being assumed that members switch into _sales mode_ when needed.
+
+## Where to start
```

Comment:
In my opinion, they should read the other sections of this document before going to the drive. I would suggest putting this at the bottom.

Reply:
I agree that it makes more sense to have that at the bottom and perhaps relabel to 'Further resources'. Also what max line length are we working with here? I assume the line length issue here is just that my IDE is set to 120, but it should be set to something lower.